BETA.md

```diff
@@ -32,8 +32,18 @@ For this step, `node` and `npm` are required.
 ## Step 3 - publish the server locally
 From the repo root run `sbt publishLocal`
 
+## Step 4 - add semanticdb-scalac compiler plugin to your project
+Some features of the server rely on the artifacts produced by [`semanticdb-scalac`](http://scalameta.org/tutorial/#semanticdb-scalac).
```

Comment:
I would move this step under the Usage section, like the `scalametaEnableCompletions` mention. Because the previous 3 steps need to be done once for all projects, while this needs to be done once for each project.

Reply:
I agree: I think a more sensible split is "Global setup" / "Per-project setup", rather than "Instructions" / "Usage"